src/CodeStyle/CSharp/Analyzers/Formatting/CSharpFormattingOptions.cs

```diff
@@ -0,0 +1,436 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Formatting
+{
+    internal static partial class CSharpFormattingOptions
+    {
+        private static readonly ImmutableArray<IOption>.Builder s_allOptionsBuilder = ImmutableArray.CreateBuilder<IOption>();
```

Comment:
?

Reply:
File was copied and minimally modified to make it compile.

Reply:
Removed this field and some others in the same file in in 513c206d0c93cd519880bb4aec73a80cc95754e7